samples/tab-channel-group-sso-quickstart/js/.env

```diff
@@ -1,4 +1,4 @@
-HTTPS=true
+HTTPS=false
```

Comment:
Do we really need to make it as false?
Will it work with https as true ?

Reply:
It will work but user need to enter ngrok URL that supports HTTPS format. Also, as in all the samples we are using http I updated to use the same for no confusion between developers.